pulsar-io/rabbitmq/src/test/java/org/apache/pulsar/io/rabbitmq/sink/RabbitMQSinkTest.java

```diff
@@ -65,7 +64,23 @@ public void TestOpenAndWriteSink() throws Exception {
         RabbitMQSink sink = new RabbitMQSink();
 
         // open should success
-        sink.open(configs, null);
```

Comment:
Use https://github.com/awaitility/awaitility it's already part of the repo , add a setup method for rabbitmq.

Reply:
@aahmed-se updated, PTAL again.